incubator/minecraft/README.md

```diff
@@ -0,0 +1,64 @@
+# Minecraft
+
+[Minecraft](https://minecraft.net/en/) Minecraft is a game about placing blocks and going on adventures.
+
+## Introduction
+
+This chart creates a single Minecraft Pod, plus Services for the Minecraft server and RCON.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`, read the [Minecraft EULA](https://account.mojang.com/documents/minecraft_eula) run:
+
+```bash
+$ helm install --name my-release \
+    --set MinecraftServer.EULA=true incubator/minecraft
```

Comment:
Can you check this works with Helm beta.2 given the following change:
https://github.com/kubernetes/helm/pull/1451

Given this will be interpreted as boolean, and the default is "FALSE" as a string.